src/utils/inner_instructions.rs

```diff
@@ -0,0 +1,57 @@
+use solana_sdk::{
+    inner_instruction::{InnerInstruction, InnerInstructionsList},
+    instruction::{CompiledInstruction, TRANSACTION_LEVEL_STACK_HEIGHT},
+    transaction_context::TransactionContext,
+};
+
+/// Pulled verbatim from `solana-svm` crate, `transaction_processor.rs`
+pub fn inner_instructions_list_from_instruction_trace(
+    transaction_context: &TransactionContext,
+) -> InnerInstructionsList {
+    debug_assert!(transaction_context
```

Comment:
Is it util to keep debug assert here?

Reply:
Yeah we can definitely remove this one. For the others, to avoid unwrapping I would probably have to change the return type to a result, which would propagate to its call site, etc.